### PR TITLE
add events & ysws to welcome message

### PR DESCRIPTION
**Pull request by aboutdavid**
image

(I map all of the channels to the test cave in development)

### CODE REVIEW

**Comment by aboutdavid** [comment]
why does it keep drafting :(

File: util/transcript.yml [hunk 1]
```diff
@@ -62,6 +62,9 @@ channels:
   hack-hour: C06SBHMQU8G
   days-of-service: C05T8E9GY64
   build-programming-languages: C06T22ZFQGP
+  cider: C073DTGENJ2
+  hamradio: C01G6UJT2RM
```
Comment (jaspermayone) [inline util/transcript.yml:66]: this won't be a ysws at the end of this month, so shouldn't be added.

**Comment by jaspermayone** [comment]
can we make sure that it doesn't do link previews?

File: util/transcript.yml [hunk 2]
```diff
@@ -146,6 +149,13 @@ house:
 
     <#${this.t('channels.ship')}> to be amazed by people just like you
 
+    Here are some active projects/events you can lurk around in or help with:
+    - <#${this.t('channels.cider')}> is a new program where you build an iOS app and you get $100 to publish it on the app store
+    - <#${this.t('channels.build-programming-languages')}> is a place you can build your own program and get mailed fudge to your doorstep
+    - <#${this.t('channels.the-boreal-express')}> is a 8-day event ran by Hack Club which you can sign up for where we'll take a train across Canada to learn new things. Invites are selective, and you must register your intrest before June 10th.
+    - <#${this.t('channels.hamradio')}> is a program where you can get your own ham radio for free by getting a <https://simple.wikipedia.org/wiki/Amateur_radio|ham radio license>.
```
Comment (jaspermayone) [inline util/transcript.yml:156]: see above comment. perhaps we replace with onboard/bin and direct to electronics?

**Review by jaspermayone — DISMISSED** [review]

**Comment by jaspermayone** [comment]
Boreal also doesn't seem to be sending correctly.

**Comment by aboutdavid** [comment]
image
fixed!

File: util/transcript.yml [hunk 3]
```diff
@@ -149,7 +150,8 @@ house:
     <#${this.t('channels.ship')}> to be amazed by people just like you
 
     Here are some active projects/events you can lurk around in or help with:
-    - <#${this.t('channels.build-programming-languages')}> is a place you can build your own program and get mailed fudge to your doorstep
+    - <#${this.t('channels.electronics')}> is a program where you can make a PCB and get $100 to print it out.
+    - <#${this.t('channels.build-programming-languages')}> is a place you can build your own program and get mailed fudge to your doorstep.
```
Comment (tregsthedev) [inline util/transcript.yml:154]: programming language

**Review by tregsthedev — CHANGES_REQUESTED** [review]
change program to programming language

File: util/transcript.yml (hunk 2 above)
Comment (aboutdavid) [reply util/transcript.yml:156]: Replaced

**Comment by jaspermayone** [comment]
@aboutdavid I would like to see some of these removed, we should be selective about what channels are recommended to new users.

**Comment by jaspermayone** [comment]
@tregsthedev, can you weigh in as well here plz?

**Comment by tregsthedev** [comment]
Will do in a bit, added to my reminders. Please ping me if I forget

**Comment by tregsthedev** [comment]
Would labeling the YSWS sections be a bad idea like:

**You ship we ship**

Here a list of activites Hack Club is running right that you can take part in! Make something related to the theme, and you can get something back (copy TBD)

Or would that be too much? _Leaning towards this_

**Comment by tregsthedev** [comment]
The Boreal express can stay!

**Comment by tregsthedev** [comment]
Remove HAM radio since it ends very soon

**Comment by jaspermayone** [comment]
I like the idea of a YSWS section!

**Comment by jaspermayone** [comment]
@aboutdavid can we see an updated message when it's ready?

**Comment by aboutdavid** [comment]
image

**Comment by jaspermayone** [comment]
can we put some form of spacer in between?

**Comment by aboutdavid** [comment]
> can we put some form of spacer in between?

wdym?

**Comment by tregsthedev** [comment]
between the two sections of the message

**Comment by tregsthedev** [comment]
@aboutdavid how does it look now?

**Review by tregsthedev — APPROVED** [review]

File: util/transcript.yml (hunk 1 above)
Comment (jaspermayone) [reply util/transcript.yml:65]: @aboutdavid we don't actually need ham radio now :)

File: util/transcript.yml [hunk 4]
```diff
@@ -147,6 +149,16 @@ house:
 
     <#${this.t('channels.ship')}> to be amazed by people just like you
 
+    Here are some active projects/events you can lurk around in or help with:
+    - <#${this.t('channels.the-boreal-express')}> is a 8-day event ran by Hack Club which you can sign up for where we'll take a train across Canada to learn new things. Invites are selective, and you must register your intrest before June 10th.
+   
+    
+    You Ship, We Ship (YSWS) are series a projects is where you make a project related to a theme, and we "ship" (send) you something, i.e. making a game and getting a game console. Here are some you can try out:
+    - <#${this.t('channels.electronics')}> (OnBoard) is a program where you can make a PCB and get $100 to print it out.
```
Comment (jaspermayone) [inline util/transcript.yml:157]: We also have bin, so this shouldnt be limited to just onboard!

File: util/transcript.yml [hunk 5]
```diff
@@ -147,6 +149,16 @@ house:
...
+    - <#${this.t('channels.build-programming-languages')}> is a place you can build your own programming language and get mailed fudge for free to your doorstep.
+    - <#${this.t('channels.sprig')}> is a game console designed by Hack Clubbers where you can make a game in JavaScript and get a free game console mailed to your doorstep.
+
+
     *Most importantly, you must introduce yourself in <#${this.t('channels.welcome')}>. There are so many intrepid travelers like yourself and they're so excited to meet you.*
```
Comment (jaspermayone) [inline util/transcript.yml:162]: I don't want the language here to be **must**, can we think of a friendlier way to suggest this?

**Review by jaspermayone — CHANGES_REQUESTED** [review]

File: util/transcript.yml (hunk 5 above)
Comment (tregsthedev) [reply util/transcript.yml:162]: You should**

File: util/transcript.yml (hunk 4 above)
Comment (tregsthedev) [reply util/transcript.yml:157]: Mention both programs and how electronics is a one-stop place for anything well eletronics related

**Pull request closed without merging**